test/print_test_stats.py

```diff
@@ -125,6 +131,29 @@ def send_report(reports):
     )
     r.raise_for_status()
 
+def send_report_to_s3(reports):
+    sha1 = os.environ.get('CIRCLE_SHA1')
+    now = datetime.datetime.utcnow().isoformat()
+    key = f'test_time/{sha1}/{now}Z.json'  # Z meaning UTC
```

Comment:
better include `CIRCLE_JOB` in here??

Reply:
yeah that's fine with me :+1: I'll do that

Reply:
Just to be clear, what is `CIRCLE_SHA1`? Is this commit ID or some random Circle job hashsum?

Reply:
@malfet it's the commit ID: https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables

Reply:
@walterddr I added the `CIRCLE_JOB` so now it comes before both the `CIRCLE_SHA1` and the timestamp; should I maybe instead change the order from `{job}/{sha1}/{now}` to `{sha1}/{job}/{now}` so that we can just use the sha as the prefix to get all test time reports within a given sha?

Reply:
see my other comment on how the data is being used. I guess a quick RFC might be a good idea to layout on the higher level what we want to achieve, what do you think? @malfet as well